Readme.md

```diff
@@ -532,6 +533,24 @@ A dashboard specific to yarn nodes resources.
 
 ![](doc/images/dashboards/Garmadon_server_overview/all.png)
 
+### Frameworks Metrics
+
+Currently only spark is supported
+
+#### Spark
```

Comment:
could you add Flink framework as well?

Reply:
Flink is for me not supported so far. We only manage it on heuristics reader and even here it seems partial.
I prefer to not add it for now and add it once we will be have something on ES and HDFS